repositories.bzl

```diff
@@ -15,8 +15,8 @@ def gapic_generator_python():
         requirements = "@gapic_generator_python//:requirements.txt",
     )
 
-    _protobuf_version = "3.21.12"
-    _protobuf_sha256 = "930c2c3b5ecc6c9c12615cf5ad93f1cd6e12d0aba862b572e076259970ac3a53"
+    _protobuf_version = "22.1"
+    _protobuf_sha256 = "0b6494b6e1a8d197f6626ca0c5aa9ab35fc1e5aa3f724787133ce4fa4aa78499"
```

Comment:
The change "3.21.12" to "22.1" indicates Protobuf has changed the "tar.gz" file names recently.

CC: @burkedavison

Reply:
Memo: https://github.com/protocolbuffers/protobuf/archive/v3.22.2.tar.gz (URL without specifying a tag) exists.